src/app/components/MessageBanner/index.test.tsx

```diff
@@ -1,97 +1,119 @@
 import React from 'react';
 import { render, screen } from '../react-testing-library-with-providers';
 import MessageBanner from '.';
-import kyrgyzBanner from './fixtures';
-
-const { summaries } = kyrgyzBanner;
-const [summary] = summaries;
+import {
+  kyrgyzMessageBannerOnePromo,
+  kyrgyzMessageBannerTwoPromos,
+} from './fixtures';
 
 describe('MessageBanner', () => {
-  it('should render a section with role region', () => {
-    render(
-      <MessageBanner
-        summaries={summaries}
-        title={kyrgyzBanner.title}
-        position={kyrgyzBanner.position}
-      />,
-    );
-    const region = screen.getByRole('region');
-    expect(region).toBeInTheDocument();
-  });
+  describe('One summary', () => {
+    const { summaries } = kyrgyzMessageBannerOnePromo;
+    const summary = summaries[0];
```

Comment:
If we wanted to, we could simplify this into a single line (unless linter complains or suggests something else)
```suggestion
    const summary = kyrgyzMessageBannerOnePromo.summaries[0];
```